e2etest/pkg/k8sclient/client.go

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package k8sclient
+
+import (
+	"os"
+
+	clientset "k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func New() *clientset.Clientset {
+	kubeconfig := os.Getenv("KUBECONFIG")
+	if kubeconfig == "" {
+		panic("KUBECONFIG not set")
+	}
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		panic(err)
+	}
+	var res *clientset.Clientset
+	res, err = clientset.NewForConfig(config)
+	if err != nil {
+		panic(err)
+	}
+	return res
+}
+
+func RestConfig() *restclient.Config {
```

Comment:
Maybe we can use `controller-runtime.GetConfigOrDie()`

import (
    ctrl "sigs.k8s.io/controller-runtime"
)

ctrl.GetConfigOrDie()

Reply:
definetely!